src/client/datascience/jupyter/jupyterExecution.ts

```diff
@@ -202,25 +199,9 @@ export class JupyterExecutionBase implements IJupyterExecution {
                         });
                     }
                     // eslint-disable-next-line no-constant-condition
-                    while (true) {
```

Comment:
Remvoing this as I don' tthink its right to change the kernel mid startup.
We should keep it consisten with Raw kernel, if startup fails we notify user and they can change it.
Found this code path was hit by a very small subset of users (previously we'd display our kernle picker & that didn't work - that was fixed in the latest relaese).

Now as part of debt, i'm just getting rid of this completely.